src/Symfony/Component/Lock/Store/PdoStore.php

```diff
@@ -330,19 +330,32 @@ private function getDriver(): string
         if ($con instanceof \PDO) {
             $this->driver = $con->getAttribute(\PDO::ATTR_DRIVER_NAME);
         } else {
-            $driver = $con->getDriver();
```

Comment:
Keeping this temp variable would make the diff a bit simpler and might ease merging from older branches.